platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -657,7 +671,7 @@ public void bind(
         stateSignerWiring.bind(stateSigner);
         pcesReplayerWiring.bind(pcesReplayer);
         pcesWriterWiring.bind(pcesWriter);
-        eventDurabilityNexusWiring.bind(eventDurabilityNexus);
+        roundDurabilityBufferWiring.bind(builder.buildRoundDurabilityBuffer());
```

Comment:
On one of my PRs I was asked to switch to the supplier binding.  Does that need to happen here?

Reply:
Yes, that should happen here as well. Fixed.